docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/local.md

```diff
@@ -1,8 +1,55 @@
 ---
-title: "Local agents"
+title: "Running a Dagster+ agent locally"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 40
 sidebar_label: "Local"
 ---
 
-# Local agents
\ No newline at end of file
+# Running a Dagster+ agent locally
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) locally. Local agents are a good way to experiment with Dagster+ locally before deploying a more scalable hybrid agent (like [Kubernetes](dagster-plus/hybrid/agents/kubernetes) or [Amazon ECS](dagster-plus/hybrid/agents/amazon-ecs-new-vpc).
+
+## Installation
+
+
+### Step 1: Install dagster-cloud
+
+```shell
+pip install dagster-cloud
+```
+
+## Step 2: Configure the agent
+
+[Generate an agent token](dagster-plus/deployment/tokens) and configure your agent
+
+```shell
+dagster-cloud configure
+```
+
+## Step 3: Run the agent
```

Comment:
This would make the guide more verbose, but it might be worth adding some note about how you'll want this process to run for a long time... and potentially ways to make that happen?

eg for the "local agent on windows" use case we've pointed folks at setting this command up via task manager to it resumes on reboot ... 

maybe we need a separate guide for how to run a prod Dagster agent on Windows

Reply:
yeah I agree -- adding something for running this as a background process would be quite useful